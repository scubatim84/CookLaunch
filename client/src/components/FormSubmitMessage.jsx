import React from 'react';
import {useStylesMain} from '../Styles';
import {Typography} from '@material-ui/core';

function FormSubmitMessage(props) {
  const classes = useStylesMain();

  return (
    <Typography id='Message' className={classes.formSubmitMessage}>
      {props.submitMessage}
    </Typography>
  );
}

export default FormSubmitMessage;
