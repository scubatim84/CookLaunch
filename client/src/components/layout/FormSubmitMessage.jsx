import React from 'react';

import {Typography} from '@material-ui/core';
import {makeStyles} from '@material-ui/core/styles';

const useStyles = makeStyles((theme) => ({
  submitMessage: {
    fontSize: '1rem',
    color: '#ff0000',
    margin: theme.spacing(2, 1, 1),
    display: 'flex',
    justifyContent: 'center',
  },
}));

function FormSubmitMessage(props) {
  const classes = useStyles();

  return (
    <Typography className={classes.submitMessage}>
      {props.submitMessage}
    </Typography>
  );
}

export default FormSubmitMessage;
