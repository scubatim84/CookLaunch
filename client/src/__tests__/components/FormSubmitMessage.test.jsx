import React from 'react';
import {shallow} from 'enzyme';
import FormSubmitMessage from '../../components/FormSubmitMessage';

describe('FormSubmitMessage', () => {
  it('Renders component without crashing', () => {
    shallow(<FormSubmitMessage />);
  });

  it('Renders submit message passed as prop to component', () => {
    const wrapper = shallow(<FormSubmitMessage submitMessage='Test message' />);

    expect(wrapper.find('#Message').equals('Test message'));
  });
});
