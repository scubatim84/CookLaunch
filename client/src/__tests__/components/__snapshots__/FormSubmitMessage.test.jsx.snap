import React from 'react';
import {shallow} from 'enzyme';
import FormSubmitMessage from '../../components/FormSubmitMessage';

describe('FormSubmitMessage', () => {
  it('Renders correctly', () => {
    const wrapper = shallow(<FormSubmitMessage />);

    expect(wrapper).toMatchSnapshot();
  });
});
